### PR TITLE
Application/#154707979

### PR DESCRIPTION
**Pull request by Mongolialady**
Pivotal Tracker Story ID: [Application/#154707979]

#### Description of Changes:

1. When mary log in, she will arrive her profile page, where she can see a list of her projects
2. when she click the name of one of her project, she will arrive the projectDetail page, where contains the detail of the project: applicant's name, message, and application date
3. when she click on the "Go Back" button, she will go back to her profile page

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ server/public/
 **/reports/
 package-lock.json
 npm-shrinkwrap.json
+yarn.lock
```
Comment (bbmoz) [inline .gitignore]: yarn.lock file should be source controlled (thus not git ignored). It allows deps to be installed with the correct version.

File: client/actions/project/index.js [hunk 2]
```diff
@@ -64,4 +64,22 @@ export default class ProjectActionCreator {
...
+  static fetchProjectById (id) {
+    console.log('show applications 3')
+    return (dispatch, getState) => {
+      dispatch(ProjectActionCreator.fetching())
+
+      try {
+        const state = getState()
+        const apiOptions = apiOptionsFromState(state)
+        const project = projectsApiClient.getProjectById(apiOptions, id)
+        console.log('fetch project by id', project)
```
Comment (bbmoz) [inline client/actions/project/index.js]: Remove `console.log`'s whenever a PR is open.

File: server/routes/controllers/projects.js [hunk 3]
```diff
@@ -52,8 +52,11 @@ export default {
   },
 
   async projectById (req, res, next, id) {
-    const project = await this.Projects.findById(id)
-
+    // const project = await this.Projects.findById(id)
```
Comment (bbmoz) [inline server/routes/controllers/projects.js]: Remove comment.

File: .gitignore (hunk 1 above)
Comment (Mongolialady) [reply .gitignore]: already revised. Please review it, thanks.

Comment (bbmoz) [reply .gitignore]: According to this, it hasn't been changed yet: https://github.com/CodeForSocialGood/calltocode.org/blob/cc61263ffb50552a36a43e84293d020f7dc7448a/.gitignore.

**Comment by Mongolialady** [comment]
sorry, changed this time.

**Pull request merged**